lib/cosmos/gui/text/ruby_editor.rb

```diff
@@ -390,6 +309,107 @@ def lineNumberAreaPaintEvent(event)
       painter.dispose
       painter = nil
     end
+
+    private
```

Comment:
This stuff can be private and it makes it clear what the API of this class actually is.